pkg/http/api/v1/handler_test.go

```diff
@@ -421,44 +423,56 @@ func TestRequestHandler_GetReady(t *testing.T) {
 }
 
 func TestRequestHandler_GetMetadata(t *testing.T) {
-	enqueuer := mock.NewEnqueuer()
-	store := mock.NewStore()
+	testCases := []struct {
+		name             string
+		mockedVersion    trivy.VersionInfo
+		mockedError      error
+		expectedHTTPCode int
+		expectedResp     string
+		expectedError    error
+	}{
+		{
+			name: "happy path",
```

Comment:
NIT: In other places I'm using kind of template for naming test cases:

Should_**do_sth**\_when\_**some\_condition**\_and\_**other\_condition**\_but\_**yet\_another\_condition**

For happy path scenarios I usually skip the **when** clause. For sad paths I'm always specifying conditions, e.g.:
- Should return version info
- Should return error when getting DB version fails

Reply:
Fixed https://github.com/aquasecurity/harbor-scanner-trivy/pull/87/commits/ca70ebfa00f3ea215b687d5d9f49c733488082fe